scripts/buildChainsJson.js

```diff
@@ -0,0 +1,104 @@
+const path = require('path');
+const { writeFile } = require('fs/promises');
+const axios = require('axios');
+
+const NOVA_CONFIG_VERSION = process.env.NOVA_CONFIG_VERSION;
+const TELENOVA_CONFIG_VERSION = process.env.TELENOVA_CONFIG_VERSION;
+const CONFIG_PATH = `chains/${TELENOVA_CONFIG_VERSION}/`;
+const TELENOVA_CONFIG_URL = `https://raw.githubusercontent.com/novasamatech/nova-utils/master/chains/${NOVA_CONFIG_VERSION}/`;
+
+const CHAINS_ENV = ['chains_dev.json', 'chains.json'];
+const ALLOWED_CHAINS = require('./data/allowedChains.json');
+
+async function getDataViaHttp(url, filePath) {
+  try {
+    const response = await axios.get(url + filePath);
```

Comment:
Node 20+ already implemented [`fetch` api](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), maybe you should use it?

Reply:
yeah, ty